tests/integration-test/src/tests/utils.rs

```diff
@@ -107,6 +121,30 @@ impl Drop for ChildGuard {
 //     Ok(bpfman_process)
 // }
 
+/// Call bpfman and check that bpfman does not panic.
+fn call_bpfman(args: Vec<&str>) -> Result<String> {
```

Comment:
Done, but went with `execute_bpfman()` instead of `execite_bpfman()`, hope you don't mind ;)

Native vs CLI vs RPC? I guess they should all be tested. The CLI will filter out some invalid options via the nature of it's syntax. Probably not this PR, so I write an Issue to think about that.